### PR TITLE
Add cask Persepolis 2.4.2

### PR DESCRIPTION
**Pull request by ibrokemypie**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/persepolis-download-manager.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'persepolis-download-manager' do
+  version '2.4.2'
+  sha256 '855756ae23ff672890f05c38bd1e146ee2d4ec218a920026ac26f286a98422a4'
+
+  # github.com/persepolisdm/persepolis was verified as official when first introduced to the cask
+  url "https://github.com/persepolisdm/persepolis/releases/download/#{version}/persepolis_#{version.major_minor_patch}_mac.dmg"
+  name 'Persepolis'
```
Comment (vitorgalvao) [inline Casks/persepolis-download-manager.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by commitay** [comment]
@ibrokemypie Use `#{version.dots_to_underscores}` instead of `#{version.major_minor_patch}`

**Comment by vitorgalvao** [comment]
> @ibrokemypie Use `#{version.dots_to_underscores}` instead of `#{version.major_minor_patch}`

As a note, @ibrokemypie, that’s what you wanted from the start. `major_minor_patch` does not output underscores.

**Comment by ibrokemypie** [comment]
is there anywhere where all of these variables are listed? Would be very helpful :P

**Comment by vitorgalvao** [comment]
[Yes](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/version.md#version-methods).

File: Casks/persepolis-download-manager.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'persepolis-download-manager' do
```
Comment (vitorgalvao) [inline Casks/persepolis-download-manager.rb]: Don’t change this back. I corrected it in accordance with the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).

**Comment by miccal** [comment]
Thank you @ibrokemypie.

**Pull request merged**